pkgs/development/tools/extism-cli/default.nix

```
@@ -1,32 +1,34 @@
-{ lib, stdenvNoCC, fetchFromGitHub, python3, makeBinaryWrapper }:
-
-stdenvNoCC.mkDerivation rec {
+{
+  pkgs ? import <nixpkgs> { system = builtins.currentSystem; }
+, lib ? pkgs.lib
+, buildGoModule ? pkgs.buildGoModule
+, fetchFromGitHub ? pkgs.fetchFromGitHub
+}:
```

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
, makeBinaryWrapper
}:
```
This is being called through `callPackage`, so you don't need to use `pkgs` as parameter.